src/specialOffer/SpecialOffer.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+import { Button, ButtonStatus } from '../button'
+import { ButtonGroup } from '../buttonGroup'
+import { Disclaimer } from '../disclaimer'
+import { Paragraph } from '../paragraph'
+import { SubHeader } from '../subHeader'
+import { StyledSpecialOffer, StyledSpecialOfferPicture } from './SpecialOffer.style'
+
+type Picture = Readonly<{
+  url: string
+  alt: string
+}>
+
+type Link = Readonly<{
+  label: string
+  url: string
+}>
+
+type Button = Readonly<{
+  label: string
+  onClick: () => void
+}>
+
+type CTALinks = [Link] | [Link, Link]
+type CTAButtons = [Button] | [Button, Button]
+
+type SpecialOfferBaseProps = Readonly<{
+  picture: Picture
+  title: string
+  description: string
+  informationLink?: Link
+}>
+
+export type SpecialOfferLinksProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaLinks: CTALinks
+  }
+>
+export type SpecialOfferButtonsProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaButtons: CTAButtons
+  }
+>
+
+function SpecialOffer(props: SpecialOfferLinksProps): JSX.Element
+function SpecialOffer(props: SpecialOfferButtonsProps): JSX.Element
```

Comment:
[Functions overload](https://krzysztofzuraw.com/blog/2019/typescript-function-overloads) to discriminate the two cases: 
- CTAs as links
- CTAs as buttons